recipe/meta.yaml

```diff
@@ -38,6 +38,7 @@ outputs:
         - setuptools
         - setuptools_scm >=4
         - setuptools_scm_git_archive
+        - wheel
         - tk  # [linux]
       run:
         - python
```

Comment:
`contourpy>=1.0.1` is required in `run`, see https://github.com/matplotlib/matplotlib/blob/318b2348f7127efc3e28aadebf762aae32b6343c/setup.py#L309

Reply:
Thanks, Serhii